links-slackbot/src/main/java/juja/microservices/links/slackbot/model/Link.java

```diff
@@ -1,4 +1,19 @@
 package juja.microservices.links.slackbot.model;
 
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+/**
+ * @author Ivan Shapovalov
+ */
+@AllArgsConstructor
+@Getter
+@EqualsAndHashCode
 public class Link {
+
+    private final String id;
+
+    private final String URL;
```

Comment:
i'd prefer a simple HashMap<String, String> rather than your model, it's always better to use simple types like String, int, boolean and BigDecimal if you wanna reach database consistency.